Sources/DeepLinking/Route+TrieRouter.swift

```diff
@@ -178,8 +178,8 @@ extension Route {
         private func parseAnnotatedRoute(_ route: URL) throws -> [Route.Component] {
 
             // use a wildcard for empty schemes/hosts, to match any scheme/host
```

Comment:
perhaps we could add a comment that host and scheme comparison should be case insensitive because of RFC3986 (same case below)